### ISSUE TITLE
Documentation needs to be updated

### ISSUE THREAD

**Issue by thomasbaldwin** [Alamofire/Alamofire#836]
Some of the code samples in the ReadMe don't work in xCode

**Comment by cnoon** [comment]
Hi @thomasbaldwin, please put yourself in our shoes. Your comment is incredibly unhelpful. 

* Which code samples are incorrect?
* Which version of the README are you looking at? 
* Which version of Xcode are you using? 

Most likely the code samples work just fine if you are using the version of Xcode the README specifies.

**Comment by thomasbaldwin** [comment]
Hi @cnoon, sorry about not being specific. I was trying to use this code in xCode 7, but it throws errors:

```swift
        Alamofire.request(.GET, "http://httpbin.org/get", parameters: ["foo": "bar"])
            .responseJSON { response in
                print(response.request)  // original URL request
                print(response.response) // URL response
                print(response.data)     // server data
                print(response.result)   // result of response serialization
                
                if let JSON = response.result.value {
                    print("JSON: \(JSON)")
                }
        }
```

<img width="1010" alt="screen shot 2015-10-04 at 5 42 19 pm" src="https://cloud.githubusercontent.com/assets/5099084/10270436/4d9edc1c-6abf-11e5-95f5-30bc6a434800.png">

I ended up getting it to work by doing something like this: http://stackoverflow.com/questions/32506703/alamofire-not-working-properly-tuple-issue-different-number-of-elements

**Comment by cristeahub** [comment]
@thomasbaldwin you are using an old version of alamofire. Update your alamofire and the examples in the documentation should work fine.

**Comment by thomasbaldwin** [comment]
I've installed using cocoa pods and am using version 2.0.2

```
platform :ios, '8.0'
use_frameworks!

target 'apollo' do
    pod 'PureLayout', '~> 3.0.1'
    pod 'SwiftyJSON', '~> 2.3.0'
    pod 'KMPlaceholderTextView', '~> 1.0.0'
    pod 'Alamofire', '~> 2.0.2'
end
```

**Comment by thomasbaldwin** [comment]
Yep I upgraded to version 2.0.2 from 2.0 and it did the trick. Thanks @cristeahub !

**Comment by thomasbaldwin** [comment]
@cristeahub I'm using cocoapods and using version 2.0.2 and still getting the error. What version do you suggest I use?

```
platform :ios, '8.0'
use_frameworks!

target 'apollo' do
    pod 'PureLayout', '~> 3.0.1'
    pod 'SwiftyJSON', '~> 2.3.0'
    pod 'KMPlaceholderTextView', '~> 1.0.0'
    pod 'Alamofire', '~> 2.0.2'
end
```

**Comment by jshier** [comment]
@thomasbaldwin If you want the documentation for 2.0.2, view the 2.0.2 tag on GitHub. Current master is for 3.0.0.

**Comment by thomasbaldwin** [comment]
Yep that's it. I just blindly installed it by copying the pod from the master readme to my podfile which was set for version 2.0. Thanks for the help @jshier :+1:

**Comment by seigel** [comment]
The readme still has reference to the install being a 2.0.x install along with the 3.0.0 examples.  FYI

![screenshot 2015-10-05 13 03 49](https://cloud.githubusercontent.com/assets/15181/10290547/080aecd8-6b62-11e5-94c6-7b19bac4413d.png)

**Comment by cnoon** [comment]
Good call @seigel! Fixed in afc8cdb3.
